.github/workflows/ci.yaml

```diff
@@ -277,20 +277,14 @@ jobs:
 
   build-and-test-fedora:
     runs-on: ubuntu-latest
-    container:
-      image: fedora:latest
+    container: fedora:latest
 
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
 
-      - name: Install rustup
```

Comment:
Fedora is comfortably ahead of our MSRV, and they're packaging the Rust compiler for a reason.